SECP-service/src/main/java/com/visucius/secp/models/Message.java

```diff
@@ -0,0 +1,64 @@
+package com.visucius.secp.models;
+
+import javax.persistence.*;
+import java.util.Date;
+
+
+@Entity
+@Table(name = "message")
+public class Message {
+
+    @Id
+    @Column(name = "message_id", unique = true, nullable = false)
+    private int messageId;
+
+    @Column(name = "timestamp")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date dateTime;
+
+    @Column(name = "body")
+    private String messageBody;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
```

Comment:
I am guessing this is a foreign key to the user. Please test this with the database to make sure it's working. Also, change user_id to id.